content/WebAuthn/WebAuthn_Developer_Guide/Overview.adoc

```diff
@@ -1,55 +1,92 @@
 
+=== Authentication ceremony
+Authentication happens locally on the authenticator and is verified by the online service. First, a user will make a request to login. The online service will send the client some data, which includes a challenge. Next, the client requests cryptographic proof of user authentication. With the user's consent, the authenticator will return an assertion. This assertion provides evidence that the authenticator is in possession of the private key by signing the client data with the private key. The server then verifies the client data has not been tampered with and uses the public key to verify the signature.
 
-==== On the Back End
+== User login flows supported
+Example FIDO2/WebAuthn authenticator flows:
 
-image::fido2_app_architecture.png[]
+* **Single factor:** Username + FIDO2 credential
+* **Second factor:** Username + password + FIDO2 credential
+* **Passwordless single factor:** FIDO2 resident key credential (a ‘resident key’ stores user data on the authenticator)
+* **Passwordless MFA:** FIDO2 resident key credential + PIN
 
-In general, a FIDO2/WebAuthn authentication architecture involves a conversation between a computing environment controlled by a Relying Party and one controlled by the user to be authenticated.
+FIDO2 is also backwards-compatible with FIDO U2F authenticators
+
+* **Single factor:** Username + FIDO U2F credential
+* **Second factor:** Username + password + FIDO U2F credential
 
-The following sections in this guide describe the basic FIDO2/WebAuthn user flows:
+== FIDO2/WebAuthn application architecture
 
-* Registration Sequence
-* Authentication Sequence
+image::fido2_app_architecture.png[]
+
+In general, a FIDO2/WebAuthn authentication architecture involves a conversation between a computing environment controlled by a Relying Party and one controlled by the user to be authenticated.
 
+=== FIDO2 Authenticator
+The user environment may consist of a client computing device with internal FIDO2 authenticator. The user may also have an external FIDO2 authenticator, such as a YubiKey, which can roam between devices. The authenticator makes credentials, generates cryptographic proof of user authentication, and manages the PIN.
 
-=== Building a WebAuthn Experience
+=== Client/Platform
+The client is the bridge between the authenticator and the RP. It implements CTAP2 and client-side WebAuthn API. The client could be a browser exposing the WebAuthn API to web applications, or an OS subsystem exposing a platform-specific FIDO API to native applications such as mobile or desktop apps.
```

Comment:
```suggestion
The client is the bridge between the authenticator and the RP. It implements CTAP2 and client-side WebAuthn API, and verifies the identity of the RP to prevent phishing attacks. The client could be a browser exposing the WebAuthn API to web applications, or an OS subsystem exposing a platform-specific FIDO API to native applications such as mobile or desktop apps.
```